app/templates/displayFilesMacro.html

```diff
@@ -1,21 +1,36 @@
 {% macro displayFiles(filepaths, titleName, deleteLink, databaseId)  %}
-<label class="mb-2"><strong>{{titleName}}</strong></label>
 <table>
+    <tr>
+        <td class="py-2" ><strong>Display</strong></td>
+        <td class="py-2 px-3"><strong>{{ titleName }}</strong></td>
```

Comment:
This might be nitpicking, but add the same amount of space after quotation marks '"' and before the greater than symbol '>'.